Readme.md

```diff
@@ -0,0 +1,48 @@
+# Okta API Access Management Sample Resource Server (ASP.NET Core)
+
+The code sample available in this repository demonstrate the use of [Okta API Access Management](http://developer.okta.com/docs/api/resources/oauth2.html) for authorizing access to a backend API using Okta's Authorization Servers.
+
+This ASP.NET Core OAuth 2.0 Resource Server will only accept calls to its TodoController if 
+1. the incoming requests contains the proper _Bearer_ OAuth token (i.e. properly validated as originating from the expected Okta organization, OpenID Connect client application and audience)
```

Comment:
1. The incoming requests contain a valid _Bearer_ OAuth token originating from the configured Okta organization

^^ I don't think we need to add in the extra stuff that you had put